src/components/search/detailed/AppSearchResults.js

```diff
@@ -117,14 +122,37 @@ export default function AppSearchResults(props) {
                 accessor: "actions",
                 Cell: ({ row }) => {
                     const original = row?.original;
+                    const partialLink = getAppListingLinkRefs(
+                        original?.system_id,
+                        original?.id
+                    )[1];
+                    const { t } = useTranslation("common");
                     return (
-                        <IconButton
-                            onClick={() => setDetailsApp(original)}
-                            size="small"
-                            color="primary"
-                        >
-                            <Info fontSize="small" />
-                        </IconButton>
+                        <Grid spacing={1}>
+                            <Grid item>
+                                <IconButton
+                                    onClick={() => setDetailsApp(original)}
+                                    size="small"
+                                    color="primary"
+                                >
+                                    <Info fontSize="small" />
+                                </IconButton>
+                            </Grid>
+                            <Grid item>
+                                <CopyLinkButton
+                                    onCopyLinkSelected={() => {
```

Comment:
Minor, but both of these buttons could use IDs for testing

Reply:
Has an id defined https://github.com/sri2k1us/sonora/blob/8e41c8d307fb7087b243c234181216e38235175f/src/components/utils/CopyLinkButton.js#L21